examples/kinematics/simple_harmonic_oscillations_of_spring.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+from sympy import solve, symbols, Eq
+from symplyphysics import Quantity, units, print_expression, convert_to
+from symplyphysics.laws.dynamics import (
+    period_of_spring_from_mass as spring_period_law,
+)
+from symplyphysics.laws.kinematic import (
+    period_from_angular_frequency as period_def,
+    displacement_in_simple_harmonic_motion as harmonic_law,
+)
+
+# Description
+## A simple harmonic oscillator consists of a block of mass 2.00 kg attached to a spring of
+## stiffness 100 N/m. When t = 1.00 s, the position and velocity of the block are x = 0.129 m
+## and v = 3.415 m/s. (a) What is the amplitude of the oscillations? What were the (b) position
+## and (c) velocity of the block at t = 0 s?
+
+block_mass, spring_stiffness, amplitude = symbols("block_mass spring_stiffness amplitude", positive=True)
+time, time1, position1, velocity1, phase_lag = symbols("time time1 position1 velocity1 phase_lag", real=True)
+
+values = {
+    block_mass: Quantity(2.00 * units.kilogram),
+    spring_stiffness: Quantity(100 * units.newton / units.meter),
+    time1: Quantity(1.00 * units.second),
+    position1: Quantity(0.129 * units.meter),
+    velocity1: Quantity(3.415 * units.meter / units.second),
+}
+
+# Find angular frequency of spring's oscillations
+
+period_expr = spring_period_law.law.rhs.subs({
+    spring_period_law.object_mass: block_mass,
+    spring_period_law.spring_elasticity: spring_stiffness,
+})
+
+angular_frequency_expr = solve(
+    period_def.law, period_def.circular_frequency
+)[0].subs(
+    period_def.period, period_expr
+)
+
+# Expression of simple harmonic oscillations of spring
+
+position_expr = harmonic_law.law.rhs.subs({
+    harmonic_law.amplitude: amplitude,
+    harmonic_law.angular_frequency: angular_frequency_expr,
+    harmonic_law.phase_lag: phase_lag,
+    harmonic_law.time: time,
+})
+
+velocity_expr = position_expr.diff(time)
+
+# (a) Amplitude of oscillations
+
+amplitude_eqn = Eq(
+    position1 ** 2 + (velocity1 / angular_frequency_expr)**2,
```

Comment:
Could you justify it?

Reply:
The reason I'm doing this is that technically I could just solve this and get the correct formulas:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/5a07b6c1-f47f-402e-8122-a1b95c05cd72)
But the solution `sympy` produces is pretty unattractive and it doesn't make sense from first sight:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/35f929c3-0d88-43f1-a718-ca8f37a85f81)
I tried to find a way for `sympy` to simplify the output but no luck